node/node.go

```diff
@@ -159,6 +162,11 @@ type TxnWithStatus struct {
 	ApplyData transactions.ApplyData
 }
 
+// SetStopAtRound changes the value of node.stopAtRound variable
+func (node *AlgorandFullNode) SetStopAtRound(rnd uint64) {
+	node.stopAtRound = rnd
```

Comment:
Somehow this needs to propagate into the catchup service. With the current implementation you would have to stop/start the node in order to resume.